src/transformers/models/llava/processing_llava.py

```diff
@@ -73,12 +88,11 @@ def __init__(
 
     def __call__(
         self,
-        text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
         images: ImageInput = None,
```

Comment:
I'm not comfortable with swapping these as it could break BC (e.g. if downstream users passes the args as position arguments)

is there a reason why we have to do this?

Reply:
Agree that it's quite ugly to handle and the standardization is causing a lot of BC issues

AFAIK some processors before VLMs used `image, text` order, so we're trying to standardize to the most commonly used format. @yonigozlan has been working on this more, and can tell if there are any other reasons